src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -109,7 +109,6 @@ import { DataPublicPluginStart } from '../../../../../plugins/data/public';
 import { DuplicateObject } from '../types';
 import { formatWorkspaceIdParams } from '../../utils';
 import { NavigationPublicPluginStart } from '../../../../navigation/public';
-import { WorkspaceObject } from '../../../../workspaces/public';
```

Comment:
why this import been removed?

Reply:
Thanks for the comment, I remove it because there isn't usage of it. If needed, I would revert it

Reply:
Thanks for spotting the error already reverted.

Reply:
Seems it should import from core now:
```js
import { WorkspaceObject } from '../../../../../core/public';
```

Reply:
This import is correct, sorry I haven't made a double check. already updated

Reply:
No problem, I guess the export location was changed in another PR, thanks for fixing this.